Signal/src/util/OWSBackupExportJob.h

```diff
@@ -0,0 +1,15 @@
+//
+//  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
+//
+
+#import "OWSBackupJob.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface OWSBackupExportJob : OWSBackupJob
```

Comment:
Using a common base class for import and export jobs simplified both considerably.